RadarSDK/RadarAPIClient.h

```diff
@@ -90,9 +93,15 @@ typedef void (^_Nullable RadarRouteAPICompletionHandler)(RadarStatus status, NSD
 
 - (void)getDistanceFromOrigin:(CLLocation *_Nonnull)origin
                   destination:(CLLocation *_Nonnull)destination
-                        modes:(RadarRouteMode)mode
+                        modes:(RadarRouteMode)modes
```

Comment:
nit:  maybe rename it to `mode:(RadarRouteMode)mode` to be consistent with the naming in the new API

Reply:
Distance API should take `modes` (can return routes for multiple modes, comma-separated), whereas mock API should take `mode` (single mode). The param is called `modes` in @blackmad's mock API PR but we should probably change to `mode` before releasing.